docs/source/windows.rst

```diff
@@ -86,3 +86,21 @@ https://python-poetry.org/docs/. Open a powershell and paste
 Sometimes, ``py`` needs to be replaced with ``python``. You can find out
 which to use by simply typing both and see which one opens the python
 interpreter terminal.
+
+
+WSL2 with vcXsrv
```

Comment:
Can you make the heading more explicit, so that we avoid abberviations in the heading. Maybe simple `using windows subsystem linux`.